includes/widgets/social-icons.php

```diff
@@ -384,16 +384,12 @@ protected function register_controls() {
 			[
 				'label' => esc_html__( 'Size', 'elementor' ),
 				'type' => Controls_Manager::SLIDER,
-				'size_units' => [ 'px', '%', 'em', 'rem', 'custom' ],
+				'size_units' => [ 'px', 'rem', 'vw', 'custom' ],
```

Comment:
@rami-elementor please add a comments for this issue

Reply:
done